modules/setting-up-an-openshift-login.adoc

```diff
@@ -0,0 +1,90 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/getting-started--with-openshift-gitops.adoc
+
+[id="setting-up-an-openshift-login_{context}"]
+= Setting up an OpenShift login
+
+.Creating a new client in Keycloak
+
+To set up an OpenShift login, you must create a new client in Keycloak:
+
+. Log into your Keycloak server.
+. Navigate to *myrealm* -> *Clients* -> then select *Create*:
+  * For *Client ID*, select *argocd*.
+  * For *Client Protocol*, select *openid-connect*.
+  * For *Root URL*, select the Argo CD route URL.
+. Configure the client:
+  * For *Access Type*, select *confidential*.
+  * For *Valid Redirect URLs*, select the callback url for your Argo CD hostname. Your callback should be `https://{hostname}/auth/callback`.
+  * Set the *Base URL* to */applications*.
+  * Save your work and then you get the *Credentials* tab. Go to this tab.
+  * For *Client Authenticator*, select *Client ID and Secret* and copy the *argocd-secret* so you can use it for your Argo CD configuration.
+
+.Configuring Argo CD OIDC
```

Comment:
Please expand OIDC the first time you use it.

Reply:
I said it means OpenID Connect...not sure if this is correct though!

Reply:
@sbose78 can you confirm, please?

Reply:
That's correct